GitVersion.yml

```diff
@@ -1,4 +1,4 @@
-next-version: 1.0
+next-version: 1.1
```

Comment:
In https://github.com/SkillsFundingAgency/das-payments-v2-common/pull/16 the major version was updated to indicate a new runtime dependency, rather than the minor, so we should probably be consistent with versioning

Reply:
Version updated to 2.0